edge/pkg/edged/edged.go

```diff
@@ -254,6 +254,14 @@ type Config struct {
 }
 
 func init() {
+	_, err := os.Stat(DefaultRootDir)
+	if !os.IsExist(err) {
+		err = os.MkdirAll(DefaultRootDir, 0755)
+		if err != nil {
+			panic(fmt.Errorf("Create %v dir error %v", DefaultRootDir, err))
```

Comment:
`Create %v dir error: %v`, add a `:` here

Reply:
@fisherxu  ok, thank you